README.md

```diff
@@ -40,6 +40,8 @@ java -jar build/libs/parasoft-demo-app-1.1.0.war --server.port=8888
 If you want to import the project into your IDE, be sure to do the following:
 1. Import the project as a Gradle project. You may need to synchronize or refresh the project after importing.
 2. Install a Lombok plugin for your IDE since the project uses Lombok.
+3. Run `build/classes` gradle task to ensure successful compilation of the project. The project uses specific technologies like gRPC, which depend on auto-generated code. it's imperative to execute this task after running the `build/clean` task as well.
```

Comment:
My suggestion:
Run the `build/classes` Gradle task to ensure the successful compilation of the project. The `build/classes` task generates auto-generated code for technologies used in the project, such as gRPC. It needs to be executed after running the `build/clean` task.